aspnet/conceptual-overview/aspnet.rst

```diff
@@ -57,19 +77,21 @@ ASP.NET Core applications are defined using a public ``Startup`` class:
       }
   }
 
-The ``ConfigureServices`` method defines the services used by your application and the ``Configure`` method is used to define what middleware makes up your request pipeline. See :doc:`/fundamentals/startup` for more details.
+- :doc:`ConfigureServices </fundamentals/startup>`  defines the services used by your app (such as the ASP.NET MVC Core framework, Entity Framework Core, Identity, etc. )
```

Comment:
**Insert parenthetical.** ...defines the services (see Services below) used by your app...